modules/ROOT/pages/to-delete-asset.adoc

```diff
@@ -9,25 +9,19 @@ If you delete the asset after seven days, you can't reuse its name, business gro
 A delete after seven days is known as a _soft delete_. After seven days, if you want to reuse the asset name,
 choose a new version number or put the asset in a different business group.
 
-To delete an asset:
-
-. Click the three-dot Settings icon at the upper right:
-+
-image::ex2-delete-asset.png[Delete Asset menu]
-+
-Alternatively, you can delete a single API version using the version menu:
-+
-image::ex2-delete-version.png[Delete a version of an asset]
-+
-Changing a RAML version? See xref:to-change-raml-version.adoc[To Change a RAML Version].
-+
-. Confirm the action when prompted, or click Cancel to exit the prompt.
+Deleting an asset version checks for any assets that are dependent on the asset version and displays a warning to prevent deletion until all the dependencies are removed.
```

Comment:
Reword to avoid the implication that "deleting an asset version" is "checking"; what is actually doing the checking?

Reply:
Would it be simpler to say that one must delete all dependencies before deleting an asset version?

Reply:
Good catch! Fixed.